vendor/engines/bulk_email/app/mailers/bulk_email/mailer.rb

```diff
@@ -14,11 +14,9 @@ def send_mail(recipient:, subject:, body:, reply_to: nil, facility:)
     end
 
     def sender
-      if @facility.try(:single_facility?)
-        "#{@facility.name} <#{default_params[:from]}>"
-      else
-        default_params[:from]
-      end
+      address = Mail::Address.new(default_params[:from])
```

Comment:
Are there any other places we might want to make use of this?

Reply:
I may want to look into one of the products DC has where they have two emails separated by a semicolon. I believe it gets used in the `reply_to` of bulk email, but I'm not sure.

I know that `Mail::Address.new` would throw away the second one. I'll put in another ticket to look into that. I'd like to get this fixed since it's preventing people from receiving email.